openfisca_us/tests/policy/baseline/gov/states/ca/tax/income/exemptions/ca_exemptions.yaml

```diff
@@ -0,0 +1,133 @@
+- name: Earns less than phase out, single, no dependents, not senior or blind
+  period: 2021
+  input:
+    people:
+      single: 
+        age: 25
+    tax_units:
+      tax_unit:
+        members: [single]
+        adjusted_gross_income: 1
+        blind_head: False
+        blind_spouse: False
+        filing_status: SINGLE
+    households:
+      household:
+        members: [single]
+        state_code: CA
+  output:
+    ca_exemptions: 129
+
+- name: Earns less than phase out, single, no dependents, senior and blind
+  period: 2021
+  input:
+    people:
+      single: 
+        age: 80
+    tax_units:
+      tax_unit:
+        members: [single]
+        adjusted_gross_income: 1
+        blind_head: True
+        blind_spouse: False
+        filing_status: SINGLE
+    households:
+      household:
+        members: [single]
+        state_code: CA
+  output:
+    ca_exemptions: 258
+
+- name: single, no dependents, senior and blind, earns two increments + 1 (2500 * 2 + 1) more than phase out (212288),
+  period: 2021
+  input:
+    people:
+      single: 
+        age: 80
+    tax_units:
+      tax_unit:
+        members: [single]
+        adjusted_gross_income: 217289
+        blind_head: True
+        blind_spouse: False
+        filing_status: SINGLE
+    households:
+      household:
+        members: [single]
+        state_code: CA
+  output:
+    ca_exemptions: 222
+
+
+- name: Earns less than phase out, married, 1 dependents, not senior or blind
+  period: 2021
+  input:
+    people:
+      parent: 
+        age: 25
+      spouse: 
+        age: 25
+      child: 
+        age: 4
+        is_tax_unit_dependent: True 
+      tax_unit:
```

Comment:
`tax_unit` is defined correctly here, but it should be under the parent `tax_units`, not `people`.